### ISSUE TITLE
Collections are always generated with the same size

### ISSUE THREAD

**Issue by mjureczko** [j-easy/easy-random#387]
Collections are always generated with the same size, presumably due to fixed seed in EasyRandomParameters.

Failing test:
https://github.com/j-easy/easy-random/compare/master...mjureczko:collection_size_not_randomized?expand=1

**Comment by benas** [comment]
Good catch, thank you for reporting this! I will check the PR #391  asap.
